generated/beats/fields.ecs.yml

```diff
@@ -2505,877 +3320,4822 @@
+    - name: module_info.authenticode.program_name
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.authenticode.publisher_link
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.compile_time
+      level: custom
+      type: date
+      description: TODO
+      default_field: false
+    - name: module_info.hash.imphash
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.hash.md5
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.hash.sha1
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.hash.sha256
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.compressed_malware_features.data_buffer
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.compressed_malware_features.decompressed_size
+      level: custom
+      type: integer
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.compressed_malware_features.encoding
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.identifier
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.prevention_threshold
+      level: custom
+      type: double
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.score
+      level: custom
+      type: double
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.threshold
+      level: custom
+      type: double
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.upx_packed
+      level: custom
+      type: boolean
+      description: TODO
+      default_field: false
+    - name: module_info.malware_score.version
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.mapped_address
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: module_info.mapped_size
+      level: custom
+      type: long
+      description: TODO
+      default_field: false
+    - name: module_info.path
+      level: custom
+      type: keyword
+      ignore_above: 1024
+   
```

Comment:
Consider changing to `is_endpoint`. Sensor is an SMP term not an Elastic Endpoint term.